### PR TITLE
RavenDB-7520

### PR DESCRIPTION
**Pull request by aviv86**

### CODE REVIEW

File: src/Raven.Client/ServerWide/Expiration/ExpirationConfiguration.cs [hunk 1]
```diff
@@ -4,15 +4,15 @@ namespace Raven.Client.ServerWide.Expiration
 {
     public class ExpirationConfiguration : IDynamicJson
     {
-        public bool Active { get; set; }
```
Comment (ayende) [inline src/Raven.Client/ServerWide/Expiration/ExpirationConfiguration.cs:7]: You have now flipped the meaning here, did you verify that this means the same.
Before, it was off by default, now it is on by default.

File: src/Raven.Client/ServerWide/Revisions/RevisionsCollectionConfiguration.cs [hunk 2]
```diff
@@ -9,15 +9,15 @@ public class RevisionsCollectionConfiguration
 
         public TimeSpan? MinimumRevisionAgeToKeep { get; set; }
 
-        public bool Active { get; set; }
+        public bool Disabled { get; set; }
```
Comment (ayende) [inline src/Raven.Client/ServerWide/Revisions/RevisionsCollectionConfiguration.cs:12]: Same

File: src/Raven.Server/Documents/Revisions/RevisionsStorage.cs [hunk 3]
```diff
@@ -86,7 +86,7 @@ public RevisionsStorage(DocumentDatabase database, Transaction tx)
                 Default = new RevisionsCollectionConfiguration
```
Comment (ayende) [inline src/Raven.Server/Documents/Revisions/RevisionsStorage.cs:86]: Need to review all creation of such things, to ensure that we don't have `Active` missing and meant to be disabled there.

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Documents/Revisions/RevisionsStorage.cs (hunk 3 above)
Comment (aviv86) [reply src/Raven.Server/Documents/Revisions/RevisionsStorage.cs:86]: found only one place where we create such things and the flag is missing, and changed it

**Pull request merged**